python/ray/tune/test/trial_scheduler_test.py

```diff
@@ -556,6 +556,15 @@ def testRemove(self):
         sched.on_trial_remove(runner, trial)  # where trial is not running
         self.assertFalse(trial in bracket._live_trials)
 
+    def testFilterNoneBracket(self):
+        sched, runner = self.schedulerSetup(100, 20)
+        # `sched' now contains None bracket, which should not be considered
+        # when calling `choose_trial_to_run'
+        sched.choose_trial_to_run(runner)
+        for hyperband in sched._get_non_empty_hyperbands():
```

Comment:
As a result of the above change, can you simply check that
1. there exists None entries in the band 
2. `choose_trial_to_run` will still work even?